chemprop/data/utils.py

```diff
@@ -539,11 +541,11 @@ def split_data(data: MoleculeDataset,
 
     elif split_type in {'cv', 'cv-no-test'}:
         if num_folds <= 1 or num_folds > len(data):
-            raise ValueError('Number of folds for cross-validation must be between 2 and len(data), inclusive.')
+            raise ValueError(f'Number of folds for cross-validation must be between 2 and the number of valid datapoints ({len(data)}), inclusive.')
 
         random = Random(0)
 
-        indices = np.repeat(np.arange(num_folds), 1 + len(data) // num_folds)[:len(data)]
+        indices = np.tile(np.arange(num_folds), 1 + len(data) // num_folds)[:len(data)]
```

Comment:
Since `indices` is shuffled in the next line, how does this affect the splitting?

Reply:
The difference between these affects the frequency that things show up when the number of data doesn't divide evenly with the number of splits. The `tile` function gives you split sizes that are all the same +/- 1, whereas the first one can differ by more. Suppose you had 10 data and three splits:
repeat -> [1,1,1,1,2,2,2,2,3,3]
tile -> [1,2,3,1,2,3,1,2,3,1]

Reply:
ah, I see, clever fix!

Reply:
Explicit credit to @swansonk14 for suggesting this function choice. I had a fix that was way more complicated and this is much better.